src/rtpghi~.c

```diff
@@ -135,11 +118,26 @@ void rtpghi_tilde_dsp(t_rtpghi_tilde *x, t_signal **sp)
 	
    post("length of c[] [bit]: %d", M * sizeof *(x->c));
    post("s_n: %d", sp[0]->s_n);
-	//post("overlap_intern:", sp[0]->s_overlap);
    
    dsp_add(rtpghi_tilde_perform, 5,x, sp[0]->s_vec, sp[1]->s_vec, sp[2]->s_vec, sp[0]->s_n);
 }
 
+void rtpghi_tilde_causal(t_rtpghi_tilde *x, t_floatarg f)
+{
+	if ((f == 0.) || (f == 1.)) {
```

Comment:
in Pd-land the typical convention for booleans is `!=0.0` (cf `[spigot]`, `[metro]`,...), so that `0.1` and `2` and `-666` are all truish.